Makefile

```diff
@@ -278,13 +276,9 @@ else
 # We skip this for the development release
 DEVEL_RELEASE_IMAGE_VERSION ?= devel
 ifneq ($(strip $(VERSION)),$(DEVEL_RELEASE_IMAGE_VERSION))
-push-$(DEFAULT_PUSH_TARGET): push-short
+push-$(DEFAULT_PUSH_TARGET):
```

Comment:
What is the purpose of this now? Can we remove this?

Reply:
Done, please check now